orsopy/fileio/base.py

```diff
@@ -0,0 +1,166 @@
+"""
+Implementation of the base classes for the ORSO header.
+"""
+
+# author: Andrew R. McCluskey (arm61)
+
+import yaml
+
+
+def noop(self, *args, **kw):
+    pass
+
+
+yaml.emitter.Emitter.process_tag = noop
+
+
+def _repr(class_to_represent):
```

Comment:
Why do you define this as a function and not a method of a base class that is inherited?